__tests__/app.test.js

```diff
@@ -67,3 +68,40 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+describe("PATCH /api/articles/:article_id", () => {
+  test("status(200),updated object is returned", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 101 })
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual({
```

Comment:
Consider using objectContaining here to avoid failing this test later after you add in the comment count test :)